docs/GettingStarted/configuration/ExporterCommittime.md

```diff
@@ -13,70 +22,223 @@ Currently we support GitHub, GitLab, BitBucket, Gitea and Azure DevOps Git servi
 
```

Comment:
I would change the previous phrase to link to the supported providers `[supported Git Providers](../Overview.md#git-providers)`

Reply:
changed